manifests/generated/kubevirt-cr.yaml.in

```diff
@@ -9,3 +9,11 @@ spec:
   configuration: {}
   customizeComponents: {}
   imagePullPolicy: {{.ImagePullPolicy}}
+  infra:
```

Comment:
Our infra components should run fine in windows containers (they can run linux inside, right?)